docs: prettier home page for the documentation portal (#309)
allinbits/cosmos-cash: docs/Networks/testnet.md

# Testnet

> A Cosmos Cash testnet is available for testing purposes, this document describes the Cosmos Cash testnet coordinates and
> the satellite projects URLs.

Reviewer: ```suggestion
A Cosmos Cash testnet is available for testing purposes.

Use these Cosmos Cash testnet coordinates and satellite project URLs:
```

## Testnet Coordinates

|              |                                                                                                     |
| ------------ | --------------------------------------------------------------------------------------------------- |
| Chain ID     | `cosmsoscash-testnet`                                                                               |
| Token  Denom | `cash`                                                                                              |
| Genesis File | [cosmos-cash.app.beta.starport.cloud/genesis](https://cosmos-cash.app.beta.starport.cloud/genesis?) |
| RPC URL      | `https://rpc.cosmos-cash.app.beta.starport.cloud:443`                                               |

### Faucet

The faucet endpoint is used to get tokens to pay for transaction gas.

|            |                                                      |
| ---------- | ---------------------------------------------------- |
| Faucet URL | `https://faucet.cosmos-cash.app.beta.starport.cloud` |

## DID Resolver

> The following are the URLs for ([DID](../Reference/GLOSSARY.md#decentralized-identifier-did)) resolution in the Cosmos Cash network

Reviewer: ```suggestion
URLs for ([DID](../Reference/GLOSSARY.md#decentralized-identifier-did)) resolution in the Cosmos Cash network.
```

|                       |                                                                         |
| --------------------- | ----------------------------------------------------------------------- |
| DID resolver driver   | https://resolver-driver.cosmos-cash.app.beta.starport.cloud/identifier/ |
| DID resolver frontend | https://uniresolver.cosmos-cash.app.beta.starport.cloud                 |

## SSI Agent Router

> The following are the URLs to connect to a SSI Agent that uses also Cosmos Cash as VDR

Reviewer: ```suggestion
URLs to connect to an SSI Agent that uses also Cosmos Cash as a verifiable data registry (VDR).
```

Reviewer: tip: use "an" for words that sound like they start "s"
